view/theme/frio/templates/admin/users.tpl

```diff
@@ -0,0 +1,273 @@
+						<td>
+							<a href="{{$baseurl}}/regmod/allow/{{$u.hash}}" title='{{$approve}}'><i class="fa fa-thumbs-up" aria-hidden="true"></i></a>
+							<a href="{{$baseurl}}/regmod/deny/{{$u.hash}}" title='{{$deny}}'><i class="fa fa-thumbs-down" aria-hidden="true"></i></a>
+						</td>
+					</tr>
+					<tr class="details">
+						<th>{{$pendingnotetext}}</th>
+						<td colspan="4">{{$u.note}}</td>
+					</tr>
+				{{/foreach}}
+					</tbody>
+				</table>
+				<div class="panel-footer text-right">
+					<button type="submit" name="page_users_deny" class="btn btn-primary"><i class="fa fa-thumbs-down" aria-hidden="true"></i> {{$deny}}</button>
+					<button type="submit" name="page_users_approve" class="btn btn-warinig"><i class="fa fa-thumbs-up" aria-hidden="true"></i> {{$approve}}</button>
+				</div>
+			{{else}}
+				<div class="panel-body text-center text-muted">{{$no_pending}}</div>
+			{{/if}}
+		</div>
+
+<!--
+	**
+	*
+	*		USERS Table
+	*
+	**
+-->
+	<div class="panel panel-default">
+		<div class="panel-heading"><h3 class="panel-title">{{$h_users}}</h3></div>
+		{{if $users}}
+
+			<table id='users' class="table table-hover">
+				<thead>
+				<tr>
+					<th></th>
+					{{foreach $th_users as $k=>$th}}
+					{{if $k < 2 || $order_users == $th.1 || ($k==5 && !in_array($order_users,[$th_users.2.1, $th_users.3.1, $th_users.4.1])) }}
+					<th>
+						<a href="{{$baseurl}}/admin/users/?o={{if $order_direction_users == "+"}}-{{/if}}{{$th.1}}">
+							{{if $order_users == $th.1}}
+								{{if $order_direction_users == "+"}}
+								&#8595;
+								{{else}}
+								&#8593;
+								{{/if}}
+							{{else}}
+								&#8597;
+							{{/if}}
+						{{$th.0}}</a>
+					</th>
+					{{/if}}
+					{{/foreach}}
+					<th>
+						<a href='#' onclick="return selectall('users_ckbx');"><i class="fa fa-check-square-o" aria-hidden="true"></i></a>
+						<a href='#' onclick="return unselectall('users_ckbx');"><i class="fa fa-square-o" aria-hidden="true"></i></a>
+					</th>
+					<th></th>
+				</tr>
+				</thead>
+				<tbody>
+				{{foreach $users as $u}}
+					<tr id="user-{{$u.uid}}">
+						<td><img class='icon' src="{{$u.micro}}" title="{{$u.nickname}}"></td>
+						<td><a href="{{$u.url}}" title="{{$u.nickname}}"> {{$u.name}}</a></td>
+						<td>{{$u.email}}</td>
+						{{if $order_users == $th_users.2.1}}
+						<td class='register_date'>{{$u.register_date}}</td>
+						{{/if}}
+
+						{{if $order_users == $th_users.3.1}}
+						<td class='login_date'>{{$u.login_date}}</td>
+						{{/if}}
+
+						{{if $order_users == $th_users.4.1}}
+						<td class='lastitem_date'>{{$u.lastitem_date}}</td>
+						{{/if}}
+
+						{{if !in_array($order_users,[$th_users.2.1, $th_users.3.1, $th_users.4.1]) }}
+						<td>{{$u.page_flags}} {{if $u.is_admin}}({{$siteadmin}}){{/if}} {{if $u.account_expired}}({{$accountexpired}}){{/if}}</td>
+						{{/if}}
+						<td>
+						{{if $u.is_deletable}}
```

Comment:
Can you please move the checkbox column all the way to the left?

Additionally, is this condition accounts for blocking as well?

Reply:
> Can you please move the checkbox column all the way to the left?
mmh I don't like to have the "expand" icon not on last column. (I don't like either the hamburger icon, I'm thinking to replace it with a "filled arrow pointing down"

> Additionally, is this condition accounts for blocking as well?
what do you mean? blocking status is shown in block/deblock action icon in contact details.
It's not optimal, but adding "(Blocked)" to that string will make it too long..
I'll leave the solution to this as an exercise for the reader..

Reply:
> Easy, leave the expand detail column on the right, move the selection column (with checkboxes) on the left.

:+1: 

> If a user can't be deleted but can be blocked, will the checkbox appear?

Oh, I read the wrong line for your comment.
I don't know. The "is_deletable" is set in `mod/admin.php`, I didn't touched that in this pr.
But if I had to guess, I would say no...

Reply:
Just to close this:
in `mod/admin.php:1751` it says

```php
   $e['is_deletable'] = (intval($e['uid']) != local_user());
```

so 'is_deletable' is more 'don't block or remove yourself'.